### PR TITLE
default item styling

### PR DESCRIPTION
**Pull request by gusreiber**

### CODE REVIEW

File: war/src/main/js/add-item.js [hunk 1]
```diff
@@ -83,6 +93,7 @@ $.when(getItems(jRoot)).done(function(data){
               class:"copy",
               description:copyDom,
               displayName:copyTitle,
+              iconFilePathPattern:'images/items/copy.png'
```
Comment (recena) [inline war/src/main/js/add-item.js:96]: Great!

File: war/src/main/js/add-item.js [hunk 2]
```diff
@@ -276,6 +284,29 @@ $.when(getItems(jRoot)).done(function(data){
...
+    function drawIcon(elem){
+      var $icn = $('<div class="icn">');
+      if (!elem.iconFilePathPattern) {
+        var name = elem.displayName;
+        var aName = name.split(' ');
+        var a = name.substring(0,1);
+        var b = ((aName.length === 1)?
+            name.substring(1,2):
+              aName[1].substring(0,1));
+        var $defIcon = $([
```
Comment (recena) [inline war/src/main/js/add-item.js:296]: It is more readable...

```
var b = ((aName.length === 1) ? name.substring(1, 2) : aName[1].substring(0, 1));
```

**Comment by recena** [comment]
@gusreiber I'm going to merge your PR...

File: war/src/main/js/add-item.js (hunk 2 above)
Comment (gusreiber) [inline war/src/main/js/add-item.js:296]: I often add line breaks on inline conditionals so I can drop a debugger on the condition, if necessary, but will 1 line it, if you prefer.

Comment (recena) [inline war/src/main/js/add-item.js:296]: @gusreiber No no, this was only an opinion. It is :+1:

**Pull request merged**